### PR TITLE
add p9dsu platform file.

### PR DESCRIPTION
**Pull request by jimsmc**
This change is [Reviewable](https://reviewable.io/reviews/open-power/skiboot/73)

### CODE REVIEW

**Comment by williamspatrick** [comment]
@mikey - I don't know if @stewart-ibm merged this already on the mailing list side or what the expectation was here.  Can you work with @jimsmc while Stewart is out?

**Comment by mikey** [comment]
@williamspatrick it's not been merged.  

@jimsmc Patch contents look good but you need to add a signed off by. Also, can you send to the mailing list, rather than a github pull request? Details here:
https://github.com/open-power/skiboot/blob/master/CONTRIBUTING.md

File: platforms/astbmc/p9dsu.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/* Copyright 2017 IBM Corp.
```
Comment (mikey) [inline platforms/astbmc/p9dsu.c]: This should probably be Supermicro, not IBM.

**Comment by jimsmc** [comment]
Close this pull request and will send the patch to the list.

**Pull request closed without merging**